src/composables/useClient.ts

```diff
@@ -60,10 +59,6 @@ export function useClient() {
         app: DEFINED_APP
       });
 
-      mixpanel.track('Propose', {
-        space: space.id
-      });
-
       return receipt;
```

Comment:
You can return from the `await client` call directly, instead of assigning to `receipt` then returning